about-page.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" />
+    <!-- <script src="https://kit.fontawesome.com/f76966036b.js" crossorigin="anonymous"></script> -->
+    <!-- CSS only -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous" />
+    <title>About the Infinity web development company</title>
+</head>
+
+<body>
+    <header class="top-header">
+        <div class="nav1 bg-dark d-none d-md-flex justify-content-end align-items-center gap-3 text-white">
+            <a href=""><i class="fa-brands">&#xf09a;</i></a>
+            <a href=""><i class="fa-brands">&#xf099;</i></a>
+            <a href="">English</a>
+            <a href="">My page</a>
+            <a href="">Logout</a>
+        </div>
+        <nav class="nav-bar">
+            <div class="logo"><a href="" class="d-flex align-items-center text-decoration-none text-black">
+                    <lord-icon src="https://cdn.lordicon.com/dfxesbyu.json" trigger="loop" delay="800"
+                        colors="primary:#242424,secondary:#ec5242" style="width:45px;height:45px">
+                    </lord-icon><span>Infinity</span>
+                </a></div>
+            <ul class="menu">
+                <li><a class="menu-item close"><img src="./assets/images/close.png" alt="close icon"></a></li>
+                <li class="menu-item"><a href="./index.html">Home</a></li>
+                <li class="menu-item"><a href="./about-page.html">About</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="header">
+        <a> <img class="humbarger" src="./assets/images/icons8-menu-50.png" width="40px" alt="Menu icon"></a>
+        <div class="headline d-flex align-items-center justify-content-center text-center">
+            <h1 class="title">Innovative, Reliable, and Effective Web Development</h1>
```

Comment:
- [ ] - [ ] I think it would be better if you update the styling of this title to match the template design e.g.: text color, font size, font weight, text transform,...
![Screenshot from 2022-11-24 00-12-01](https://user-images.githubusercontent.com/83042742/203654660-267f4bd5-b566-4fa2-b0ec-94b4aa8730bb.png)